client/lib/signup/step-actions/index.js

```diff
@@ -205,7 +202,7 @@ export function createSiteWithCart( callback, dependencies, stepData, reduxStore
 
 	if ( shouldSkipDomainStep || shouldHideFreePlan || shouldHideDomainStep ) {
 		newSiteParams.blog_name =
-			get( user.get(), 'username' ) ||
+			get( user().get(), 'username' ) ||
```

Comment:
The `lodash/get` here is not needed, by virtue of `false` being an object. If you don't want to (ab)use that (which I understand 🙂), optional chaining would still be a more modern option.

Reply:
Switched to optional chaining 🙂